CITATION.cff

```diff
@@ -0,0 +1,32 @@
+cff-version: 1.2.0
+title: GeoCAT-applications
+message: >-
+  If you use this resource, please cite it using the
+  metadata from this file.
+authors:
+  - given-names: Katelyn
+    family-names: FitzGerald
+    email: katelynw@ucar.edu
+    affiliation: NSF NCAR
+    orcid: 'https://orcid.org/0000-0003-4184-1917'
+  - given-names: Julia
+    family-names: Kent
+    email: jkent@ucar.edu
+    affiliation: NSF NCAR
+    orcid: 'https://orcid.org/0000-0002-5611-8986'
+  - given-names: Andy
+    family-names: McKeen
+    affiliation: 'Vermont State University, NSF NCAR'
```

Comment:
I also made a new orchid, `orchid: https://orcid.org/0009-0005-2551-8118`
And for email I guess `email: andymckeen648@gmail.com` because my UCAR and school emails will be useless pretty soon

Reply:
Done! Except the email per other comments